Shared Framework Code/API/NightscoutAppDataManager.swift

```
@@ -49,48 +62,58 @@ public func quickFetch(site: Site, handler: (returnedSite: Site, error: Nightsco
+            nsAPI.fetchCalibrations(numberOfCalsNeeded, completetion: { (calibrations, errorCode) -> Void in
                 errorToReturn = errorCode
                 
-                print("STEP 4:      GET Sever CALs/Calibrations for site: \(site.url)")
-                let numberOfCalsNeeded = ((Constants.EntryCount.NumberForComplication * 5) / 60) / 12 + 1
-                nsAPI.fetchCalibrations(numberOfCalsNeeded, completetion: { (calibrations, errorCode) -> Void in
-                    errorToReturn = errorCode
-                    
-                    guard let calibrations = calibrations else {
-                        return
-                    }
-                    
-                    let cals = calibrations.sort{(item1:Entry, item2:Entry) -> Bool in
-                        item1.date.compare(item2.date) == .OrderedDescending
-                        }.flatMap { $0.cal }
-                    
-                    site.calibrations = cals
-                    
-                    print("STEP 5:      Generate Timeline data for Complication for site: \(site.url)")
-                    let complicationModels = generateComplicationModels(forSite: site, calibrations: site.calibrations)
-                    site.complicationModels = complicationModels
-                    
+                guard let calibrations = calibrations else {
                     NSOperationQueue.mainQueue().addOperationWithBlock({ () -> Void in
-                        print("COMPLETE:    All network operations are complete for site: \(site.url)")
-                        print("DURATION:    The entire process took: \(NSDate().timeIntervalSinceDate(startDate))")
-                        print("STEP 6:      Return Handler to main thread.")
-                        handler(returnedSite: site, error: errorToReturn)
+                        handler(returnedSite: site, error: errorCode)
```

@rodericj, Not all devices or users of nightscout have or use calibrations. The UI should be hiding portions of the UI that require calibrations, so things should fail silently. At least I think so.